services/terraform/terraform-module-downloads.service.js

```diff
@@ -0,0 +1,60 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import { pathParams } from '../index.js'
+import {
+  BaseTerraformService,
+  description,
+  intervalMap,
+} from './terraform-base.js'
+
+export default class TerraformModuleDownloads extends BaseTerraformService {
+  static category = 'downloads'
+
+  static route = {
+    base: 'terraform/module/downloads',
```

Comment:
If you have a look over all the other downloads badges, we just use `serviceName/:interval(dw|dm|dy)...` or whatever. We can remove the `/downloads` part from the route base here.